clippy_lints/src/doc/needless_doctest_main.rs

```diff
@@ -86,13 +84,22 @@ pub fn check(
                                     // This main function should not be linted, we're done
                                     eligible = false;
                                 }
+                                // Return early if we're in an ignore codeblock, as
+                                // test_attr_in_doctest
+                                // won't do anything useful in these cases and we already have our
+                                // problematic function.
```

Comment:
Is this an outdated comment? There's no early return in this arm (or is this also talking about the early return a few lines down?)

Reply:
Yep, it was an outdated comment, removed!